### ISSUE TITLE
"Valet secure" writes bad config and causes daemon/Caddy to crash.

### ISSUE THREAD

**Issue by themrwilliams** [laravel/valet#117]
Beginning...

Dnsmasq and PHP are installed via Homebrew:

➜  ~ ls -l /Library/LaunchDaemons/
total 24
-rw-------  1 root  wheel  576 Aug 17 15:09 homebrew.mxcl.dnsmasq.plist
-rw-------  1 root  wheel  705 Aug 17 15:33 homebrew.mxcl.php70.plist
lrwxr-xr-x  1 root  wheel   76 Apr 14 10:54 org.virtualbox.startup.plist -> ../Application Support/VirtualBox/LaunchDaemons/org.virtualbox.startup.plist

Valet installed globally via Computer.

➜  ~ composer global show -i | grep valet
Changed current directory to /Users/mwilliams/.composer
laravel/valet                       v1.1.18 A more enjoyable local development experience for Mac.

➜  ~ which valet
/Users/mwilliams/.composer/vendor/bin/valet

➜  ~ valet install

Valet installed successfully!

➜  ~ cd Sites
➜  Sites laravel new blog

... lots of typical stuff that happens when you make a new Laravel thing ...

➜  blog valet link
A [blog] symbolic link has been created in [/Users/mwilliams/.valet/Sites/blog].

http://blog.dev/ works! :)

bash-3.2# launchctl list | grep valet | awk '{ print $2 }'
0

Valet daemon is working too, of course.

➜  blog valet secure
The [blog.dev] site has been secured with a fresh TLS certificate.

bash-3.2# launchctl list | grep valet | awk '{ print $2 }'
1

Oh noes!  Valet daemon is a gonner :(

bash-3.2# cat /tmp/com.laravel.valetServer.err
2016/08/17 15:11:05 open VALET_HOME_PATH/Log/access.log: no such file or directory

More info:

➜  Sites which php
/usr/local/bin/php
➜  Sites php -v
PHP 7.0.9 (cli) (built: Jul 21 2016 14:50:47) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
➜  Sites

➜  Sites ps aux | egrep -i '(valet|caddy|php|dnsmas)'
nobody          98615   0.0  0.0  2445320    924   ??  Ss    3:43PM   0:00.01 /usr/local/opt/dnsmasq/sbin/dnsmasq --keep-in-foreground -C /usr/local/etc/dnsmasq.conf
mwilliams       99796   0.0  0.0  2432792    588 s004  R+    3:48PM   0:00.00 egrep -i (valet|caddy|php|dnsmas)
mwilliams       99746   0.0  0.0  2482564    668   ??  S     3:47PM   0:00.00 /usr/local/opt/php70/sbin/php-fpm --fpm-config /usr/local/etc/php/7.0/php-fpm.conf
mwilliams       99745   0.0  0.0  2482564    672   ??  S     3:47PM   0:00.00 /usr/local/opt/php70/sbin/php-fpm --fpm-config /usr/local/etc/php/7.0/php-fpm.conf
root            99741   0.0  0.0  2483588   7772   ??  Ss    3:47PM   0:00.03 /usr/local/opt/php70/sbin/php-fpm --fpm-config /usr/local/etc/php/7.0/php-fpm.conf

Host machine is running OSX 10.11.4.

```
➜  .valet cat Caddyfile
import /Users/mwilliams/.valet/Caddy/*

:80 {
    fastcgi / 127.0.0.1:9000 php {
        index server.php
    }

    rewrite {
        to /server.php?{query}
    }

    log /Users/mwilliams/.valet/Log/access.log {
        rotate {
            size 10
            age 3
            keep 1
        }
    }

    errors {
        log /Users/mwilliams/.valet/Log/error.log {
            size 10
            age 3
            keep 1
         }
    }
}

➜  .valet cat config.json
{
    "domain": "dev",
    "paths": [
        "/Users/mwilliams/.valet/Sites"
    ]
}

➜  .valet cat Caddy/blog.dev
http://blog.dev:80 {
    redir https://{host}{uri}
}

https://blog.dev:443 {
    tls /Users/mwilliams/.valet/Certificates/blog.dev.crt /Users/mwilliams/.valet/Certificates/blog.dev.key

    fastcgi / 127.0.0.1:9000 php {
        index server.php
    }

    rewrite {
        to /server.php?{query}
    }

    log VALET_HOME_PATH/Log/access.log {       <------- OH NOES?!
        rotate {
            size 10
            age 3
            keep 1
        }
    }

    errors {
        log VALET_HOME_PATH/Log/error.log {    <------- OH NOES?!
            size 10
            age 3
            keep 1
         }
    }
}
```

**Comment by adamwathan** [comment]
Fixed and tagged 1.1.19, thank you!

**Comment by ChaseGiunta** [comment]
It took me way too long to realize that I needed to delete my `~/.valet/` folder and reinstall valet after encountering this bug and updating to latest version. 

Be sure to do that, as something in the config is obviously bad and updating Valet will not subsequently replace the `~/.valet/` or fix the bad config.
